web/src/components/Section.js

```diff
@@ -48,30 +47,44 @@ Section.defaultProps = {
   isNumbered: false
 };
 
-const Subsection = ({ children, id, nested, open, resource }) => {
+const Subsection = ({ children, id, nested, resource }) => {
   const title = t([resource, 'title'], { defaultValue: '' });
 
   return (
-    <Collapsible id={id} title={title} open={open} nested={nested}>
+    <Fragment>
+      {!nested &&
+        <h3
+          id={id}
+          className='subsection--title ds-h3'
+        >
+          {title}
+        </h3>
+      }
+      {nested &&
+        <h4
+          id={id}
+          className='ds-h3'
+        >
+          {title}
+        </h4>
```

Comment:
I'm not crazy about this way of doing this -- as far as I can tell, the activity details are the only thing that need the `h4` version. I'd rather just have them not call the subsection, since this seems to just be rendering the children now with very little other benefit.

Reply:
👍 This sounds like something to address when we start tackling converting individual sections to the design system.